### PR TITLE
Adding data changes

### PR DESCRIPTION
**Pull request by noahmanger**
Adding explanation of what we changed for the upcoming release to the July 7 release notes. 

I'd love @jeanninehunter 's 👓  for copy, and also @LindsayYoung to verify I'm explaining the API changes correctly (and please let me know if there's more we should announce).

Also, @PaulClark2 and @amykort, how does this look to you?

cc @jenniferthibault and @emileighoutlaw and @porta-antiporta for visibility.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
Looks good I added 2 little things

File: release_notes/release_notes_2016-07-07.md [hunk 1]
```diff
@@ -22,8 +22,10 @@
 
 ## Calendar
 - Filing deadlines now include links to the report notices page on fec.gov
+- Links to reporting pages for reporting dates
```
Comment (noahmanger) [inline release_notes/release_notes_2016-07-07.md]: Isn't this and the line above the same thing?

File: release_notes/release_notes_2016-07-07.md [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
 
 ## Legal resources
 - A new, searchable legal resources section was added that includes the latest annual edition of the Code of Federal Regulations Title 11.  We are working to add Statutes and Advisory Opinions in a later release.
+
+## Registration and reporting
+- Each registrant-type row on beta.fec.gov/registration-and-reporting now includes a link to the filing deadlines for that type
+
+## Calendar
+- Filing deadlines now include links to the report notices page on fec.gov
+- Links to reporting pages for reporting dates
```
Comment (jenniferthibault) [inline release_notes/release_notes_2016-07-07.md]: Would love to make this a complete sentence, or somehow more descriptive of what it means/why it matters. I have no idea.

File: release_notes/release_notes_2016-07-07.md [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+
+## API
+- Added a new endpoint to find the Reports and Analysis Division analyst for a particular committee ID
+- Added a new endpoint to view financial totals for all PACs: `/totals/pac`
+- Added a new endpoint to view financial totals for all party committees: `/totals/party/`
+- Upgraded data source for candidate and committee totals
```
Comment (jenniferthibault) [inline release_notes/release_notes_2016-07-07.md]: Same here—what does this mean?

File: release_notes/release_notes_2016-07-07.md (hunk 2 above)
Comment (emileighoutlaw) [inline release_notes/release_notes_2016-07-07.md]: Tagging in @jeanninehunter to address!

File: release_notes/release_notes_2016-07-07.md [hunk 4]
```diff
@@ -1,7 +1,31 @@
...
+
+## Registration and reporting
+- Each registrant-type row on beta.fec.gov/registration-and-reporting now includes a link to the filing deadlines for that type
+
+## Calendar
+- Filing deadlines now include links to the report notices page on fec.gov
+- Links to reporting pages for reporting dates
+
+## API
+- Added a new endpoint to find the Reports and Analysis Division analyst for a particular committee ID
```
Comment (emileighoutlaw) [inline release_notes/release_notes_2016-07-07.md]: I believe the name of this division is `Reports Analysis Division`

File: release_notes/release_notes_2016-07-07.md [hunk 5]
```diff
@@ -1,31 +1,35 @@
...
 ## Registration and reporting
-- Each registrant-type row on beta.fec.gov/registration-and-reporting now includes a link to the filing deadlines for that type
+- Added registration and reporting essentials for nonconnected committees
+- Linked beta.fec.gov/registration-and-reporting to calendar filing deadlines for each type of registrant
 
 ## Calendar
-- Filing deadlines now include links to the report notices page on fec.gov
-- Links to reporting pages for reporting dates
+- Added links to the fec.gov report notices for filing deadlines 
+- Linked to reporting pages for reporting dates
```
Comment (noahmanger) [inline release_notes/release_notes_2016-07-07.md]: I'm still confused about what this means. Are these two points different? @LindsayYoung

Comment (emileighoutlaw) [inline release_notes/release_notes_2016-07-07.md]: +1 to this comment

**Pull request merged**